Data/core_rulebook/cr_abilities.lua

```diff
@@ -2069,12 +2069,12 @@ DefineAbility({
   Category="Special Ability",
   AllowMultiple=true,
   Stackable=false,
-  Chooses={
-    ChooseSkill(function (character, skill)
-      return (skill.Type == "Base")
+  Choice={
+    Choose=ChooseSkill(function (character, skill)
+      return ((skill.IsType("Base")))
     end),
   },
-  Selections="1",
+  Selections=nil,
```

Comment:
What does "nil" mean? Also, should the other things be numbers? I'm scared that they are string.  "Numer as strings" make parsing a chore, because it means that we need to handle "what if this isn't a number"

Reply:
nill is just LUA null. These can be "formulas" too so its a string for now.

Reply:
Hrm... I wonder if things we know are formulas should be go from `value` to `Formula(value)`, so that it's clear they A) are formulas and B) are easier to handle with the parser.  String should just be strings.

Reply:
I'll do that.